payments/providers/__init__.py

```diff
@@ -1,15 +1,21 @@
-from typing import Optional
-
-import environ
 from django.conf import settings
 from django.http import HttpRequest
 from django.utils.module_loading import import_string
 
-# imported here so that we can refer to it like 'payments.providers.BamboraPayformProvider'
-from .bambora_payform import BamboraPayformProvider  # noqa
+from utils.config import get_config_from_env
+
+from .bambora_payform import BamboraPayformProvider
 from .base import PaymentProvider
+from .talpa_ecom import TalpaEComProvider
+
+_provider_class: PaymentProvider
 
-_provider_class: Optional[PaymentProvider] = None
+__all__ = [
+    "BamboraPayformProvider",
```

Comment:
Can we just remove Bambora code from the codebase?

Reply:
For now, we should keep it because we're not gonna switch to Talpa _yet_. Also, it would be maybe nice to keep it for documentation purposes, in case some other service needs to look at it. But maybe worth removing once we've fully migrated to Talpa.

Reply:
ok!